---
uint32 speed  # The current sidebrush speed level
